spec/services/forest_liana/permissions_getter_spec.rb

```diff
@@ -0,0 +1,65 @@
+module ForestLiana
+  describe PermissionsGetter do
+    describe '#get_permissions_api_route' do
+      it 'should respond with the v3 permissions route' do
+        expect(described_class.get_permissions_api_route).to eq '/liana/v3/permissions'
+      end
+    end
+
+    describe '#get_permissions_for_rendering' do
+      let(:rendering_id) { 34 }
+      let(:liana_permissions_url) { 'https://api.forestadmin.com/liana/v3/permissions' }
+      let(:liana_permissions_api_call_response) { instance_double(HTTParty::Response) }
+      let(:expected_query_parameters) {
+        {
+          :headers => {
+            "Content-Type" => "application/json",
+            "forest-secret-key" => "env_secret_test"
+          },
+          :query => { "renderingId" => rendering_id }
+        }
+      }
+
+      before do
+        allow(HTTParty).to receive(:get).and_return(liana_permissions_api_call_response)
+        allow(liana_permissions_api_call_response).to receive(:response).and_return(liana_permissions_api_call_response_content)
+        allow(liana_permissions_api_call_response_content).to receive(:body).and_return(liana_permissions_api_call_response_content_body)
+      end
+
+      describe 'when the API returns a success' do
+        let(:liana_permissions_api_call_response_content) { Net::HTTPOK.new({}, 200, liana_permissions_api_call_response_content_body) }
+        let(:liana_permissions_api_call_response_content_body) { '{"test": true}' }
+        let(:expected_parsed_result) { { "test" => true } }
+
+        it 'should call the API with correct URL' do
+          described_class.get_permissions_for_rendering(rendering_id)
```

Comment:
I see that you always use `described_class` wich does not give much information. What is the way to go ? Use `described_class` are directly `ForestLiana::PermissionsGetter::` ?

Reply:
described_class is the preferred way for rspec tests, it refers to the provided class at the start of the file